test/01-constructor.test.js

```diff
@@ -0,0 +1,27 @@
+const MyToken = artifacts.require("MyToken");
+
+contract("Constructor", accounts => {
+
+    before(async () => {
+        contractInstance = await MyToken.new();
+    });
+
+    it("gives the owner of the token 1M tokens with 6 decimals", async () => {
+        let balance = await contractInstance.balanceOf(accounts[0]);
+        balance = web3.utils.fromWei(balance, 'ether');
+        assert.equal(balance, '0.000001', "Balance should be 1M tokens with 6 decimals for contract creator")
```

Comment:
Use BigNumber to equalize. 
There is an example in the "internship" channel.
For fromWei you can use next parameter: web3.utils.fromWei(balance, 'mwei'); ( or other, please recheck)
https://web3js.readthedocs.io/en/v1.2.11/web3-utils.html#fromwei